securedrop_client/storage.py

```diff
@@ -422,3 +422,7 @@ def source_exists(session: Session, source_uuid: str) -> bool:
         return True
     except NoResultFound:
         return False
+
+
+def get_file(session: Session, file_uuid: str) -> File:
```

Comment:
this PR adds two new functions (well one is a method on the `Controller`) both called `get_file` - this one in storage.py looks unnecessary?

Reply:
> this PR adds two new functions (well one is a method on the `Controller`) both called `get_file` - this one in storage.py looks unnecessary?

in order to remove querying the db from the gui, i added get_file, as you can see here: https://github.com/freedomofpress/securedrop-client/pull/374/commits/e21084ea3fbbe14b86ecbc3a653f19171f1c8cca

The controller::get_file calls storage::get_file, where it looks like all of our db queries are maintained: https://github.com/freedomofpress/securedrop-client/pull/374/files#diff-473509107be52a27c95435c68c00c7d1R625

Reply:
ahh gotcha